samples/core/xgboost_training_cm/xgboost_training_cm.py

```diff
@@ -201,86 +226,94 @@ def dataproc_predict_op(
     description='A trainer that does end-to-end distributed training for XGBoost models.'
 )
 def xgb_train_pipeline(
-    output,
-    project,
+    output='gs://guideline_example_bucket',
+    project='ml-pipeline-dogfood',
+    cluster_name='xgb-{{workflow.name}}',
     region='us-central1',
     train_data='gs://ml-pipeline-playground/sfpd/train.csv',
     eval_data='gs://ml-pipeline-playground/sfpd/eval.csv',
     schema='gs://ml-pipeline-playground/sfpd/schema.json',
     target='resolution',
-    rounds=200,
+    rounds=5,
     workers=2,
     true_label='ACTION',
 ):
     output_template = str(output) + '/' + dsl.EXECUTION_ID_PLACEHOLDER + '/data'
 
-    delete_cluster_op = dataproc_delete_cluster_op(
-        project,
-        region
-    ).apply(gcp.use_gcp_secret('user-gcp-sa'))
+    # Current GCP pyspark/spark op do not provide outputs as return values, instead,
+    # we need to use strings to pass the uri around.
+    analyze_output = output_template
+    transform_output_train = os.path.join(output_template, 'train', 'part-*')
+    transform_output_eval = os.path.join(output_template, 'eval', 'part-*')
+    train_output = os.path.join(output_template, 'train_output')
```

Comment:
I'm not sure this manual path modification is needed.
Also this might not work since the placeholder resolving happens at runtime.

Reply:
Solved offline. Specifically RUN_ID_PLACEHOLDER can work here, but not EXECUTION_ID_PLACEHOLDER